catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImpl.java

```diff
@@ -39,17 +39,23 @@
 import org.xml.sax.SAXException;
 
 public class Gml3ToWktImpl implements Gml3ToWkt {
+
   private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
 
   private static final String EPSG_4326 = "EPSG:4326";
 
-  private final Parser parser;
+  private final ThreadLocal<Parser> parser;
 
   private static final ThreadLocal<WKTWriter> WKT_WRITER = ThreadLocal.withInitial(WKTWriter::new);
 
   public Gml3ToWktImpl(Configuration gmlConfiguration) {
-    parser = new Parser(gmlConfiguration);
-    parser.setStrict(false);
+    parser =
+        ThreadLocal.withInitial(
+            () -> {
+              Parser parser = new Parser(gmlConfiguration);
```

Comment:
I agree that it is more minor than major. However, either you suppress it or change the name. Since suppressing it means modifying the code, might as well rename it. The only downside, to me, would be in a large method where someone might loose track of the fact that they are not referencing the class attribute but the local variable instead.

Reply:
I'll just change it, then!